README.md

```diff
@@ -8,12 +8,28 @@ NeonBee is an open source reactive dataflow engine, a data stream processing fra
+
+- **Need an API? Get an API**: As briefly mentioned in the previous points already, Vert.x by default does not provide you with a standardized API endpoint. If you need one, build one, or use any of the Vert.x plugins to get one configured for you. How does it interact with other endpoints, is it properly integrated, e.g., into the event bus, so that consumption of thousands of requests from the endpoint remains scalable? Well, something that will depend heavily on your implementation of the server / endpoint. As a dataflow engine, we have been under the assumption that data processing means nothing, if you cannot expose your result. Thus, NeonBee comes with build-in endpoints and an extensible end-point concept. Meaning that you can get a REST and / or OData endpoint simply by configuring it. It runs integrated with the concept of `DataVerticle` and `EntityVerticle`, meaning that everything that you expose via a `DataVerticle` can also (without one line of additional code), be exposed via a REST endpoint if you choose to do so. Generally saying, you know `Servlets`, right? Well, you didn't have to take care that a server existed to expose the servlet back in the days, why should you know? Even more so, NeonBee provides you with a concept of structured and unstructured endpoints, meaning that you can expose a full data model, of which the data is provided by an independently scalable set of verticles "under the hood". Again, one less thing you have to worry about!
```

Comment:
Generally saying, you know `Servlets`, right? Well, you didn't have to take care that a server existed to expose the servlet back in the days, why should you know?

This sentence sounds strange too me. But maybe it is because my English is not the best.

Reply:
I have changed the sentence completly, thanks Pascal! Also feel free to add any additional itms if you have more ideas what is better to use NeoNbee for :)